app/src/main/java/com/gh4a/fragment/RepositoryFragment.java

```diff
@@ -125,6 +89,8 @@ public void onDestroyView() {
         super.onDestroyView();
         mImageGetter.destroy();
         mImageGetter = null;
+
+        for(Disposable d : mSubscriptions) if(d != null) d.dispose(); // clean up rx
```

Comment:
Try to avoid one liners. It's better to always add braces to loops, ifs, etc. Also add space before opening brace, and the comment is useless 🙂